### ISSUE TITLE
Golden Sun GBA - Overworld map glitches out

### ISSUE THREAD

**Issue by DeezY48** [skylersaleh/SkyEmu#439]

**Comment by skylersaleh** [comment]
Can you add a picture and save state?
Also, are you using an official GBA BIOS?

**Comment by DeezY48** [comment]
Yeah, for sure! Here is the screenshot and the save state. Also i don't think i am using an official GBA BIOS, its red and has an X so that's probably the reason but i also don't know how to get one. 
[Golden Sun (USA, Europe).zip](https://github.com/skylersaleh/SkyEmu/files/14376767/Golden.Sun.USA.Europe.zip)
![Screenshot 2024-02-22 10 47 29 AM](https://github.com/skylersaleh/SkyEmu/assets/160884026/652acdf0-13ad-4e02-83f4-da6f51a20d29)
![Screenshot 2024-02-22 11 45 25 AM](https://github.com/skylersaleh/SkyEmu/assets/160884026/0d80d365-66fc-4c06-b346-36785acc9e1e)

**Comment by Hambaka** [comment]
It seems that even with the official GBA BIOS, this issue still exists. (latest Windows dev build)
![SkyEmu_GS](https://github.com/skylersaleh/SkyEmu/assets/20476755/48df1571-ed42-4a62-ab9e-f428940f379c)

**Comment by DeezY48** [comment]
Yeah, it took me a while but when i got the official GBA BIOS, the problem was still there.

**Comment by DeezY48** [comment]
I finally got a gba BIOs but the issue still wasn't  resolved
